generic/tests/netperf.py

```diff
@@ -482,6 +482,7 @@ def count_interrupt(name):
         sum = 0
         intr = []
         stat = ssh_cmd(server_ctl, "cat /proc/interrupts |grep %s" % name)
+        stat = stat.strip().split("\n")[-1]
         for i in stat.strip().split("\n"):
```

Comment:
Hit the follows error with remove them. 

     sum += int(i.split()[cpu + 1])
     IndexError: list index out of range

So, strip () and split () still are needed.